addons/hr_recruitment/static/src/scss/hr_job.scss

```diff
@@ -6,33 +6,6 @@
                 min-height: 220px;
             }
         }
-
-        .text_top_padding{
-            padding-top: 0.375rem;
-        }
-
-        .o_primary {
-            display: flex !important;
-            align-items: center;
-            justify-content: space-between;
-            padding-right: 24px !important;
-
-            .fa {
-                margin-left: 0;
-            }
-        }
-
-        .o_job_activities {
-            list-style-type: none;
-        }
-
-        .o_kanban_card_header_title {
```

Comment:
commit: 6f5722acb43ec48f07ef9d8a54416e96c28a212a
Good cleaning, care that this line was probably used to align the buttons vertically between cards with the ones having the text-muted and those without. I'm fine with it, but it creates a visual change.